templates/configmap-env.yaml

```diff
@@ -34,20 +34,17 @@ data:
   DEFAULT_LOCALE: {{ .Values.mastodon.locale }}
   {{- if .Values.elasticsearch.enabled }}
   ES_ENABLED: "true"
-  ES_PRESET: {{ .Values.elasticsearch.preset | default "single_node_cluster" | quote }}
   ES_HOST: {{ template "mastodon.elasticsearch.fullname" . }}-master-hl
   ES_PORT: "9200"
   {{- else if .Values.elasticsearch.hostname }}
   ES_ENABLED: "true"
+  ES_PRESET: {{ .Values.elasticsearch.preset | default "single_node_cluster" | quote }}
```

Comment:
You should be able to define this variable is ES is enabled, wether it is through `elasticsearch.enabled` or with `elasticsearch.hostname` defined

Reply:
Yeah that makes sense. Fixed.